### ISSUE TITLE
You don't have a CHANGES file.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gyc2432968/python-twitter#179]
```
What steps will reproduce the problem?
1. Attempt to easy_install python-twitter 
2. Observe error message regarding missing CHANGES file
3. RAGE

What is the expected output? What do you see instead?
I would like to easy_install python-twitter.  Instead, I see an unending pit of 
despair and pain.

What version of the product are you using? On what operating system?
python-twitter 0.8 on FreeBSD

Please provide any additional information below.
thank you for your time


error: CHANGES: No such file or directory
An error occured when trying to install python-twitter 0.8. Look above this 
message for any errors that were output by easy_install.

```

Original issue reported on code.google.com by `karmaf...@gmail.com` on 21 Dec 2010 at 6:21

**Comment by GoogleCodeExporter** [comment]
```
This is only a problem in the release tarball, by the way.

Also the release tarball has Mac resource forks in it.
```

Original comment by `karmaf...@gmail.com` on 21 Dec 2010 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
ugh! this is what I get for trying to get this out the door after a week of 
being sick...

I'll fix these issues and rollout a 0.8.1 

thanks
```

Original comment by `bear42` on 21 Dec 2010 at 11:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
who knew that python 2.6+ ignores the MANIFEST.in file (or maybe i'm using it 
wrong.. ?)

Recreated the sdist tarball using python2.5 and verified that the extra files 
are now present
```

Original comment by `bear42` on 24 Dec 2010 at 4:54
- Changed state: **Fixed**
